UmbracoLinqPad.Gateway/UmbracoLinqPad.Gateway.csproj

```diff
@@ -20,6 +20,7 @@
     <DefineConstants>DEBUG;TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
+    <PlatformTarget>x86</PlatformTarget>
```

Comment:
No, but it got into the commit somehow. Just delete please

Met vriendelijke groet,

Vincent Baaij
________________________________
From: Shannon Deminick<mailto:notifications@github.com>
Sent: ‎14-‎4-‎2015 00:28
To: Shazwazza/UmbracoLinqPadDriver<mailto:UmbracoLinqPadDriver@noreply.github.com>
Cc: Vincent Baaij<mailto:vnbaaij@outlook.com>
Subject: Re: [UmbracoLinqPadDriver] Workaround for bin folder containing not .net dll's (#1)

> @@ -20,6 +20,7 @@
>      <DefineConstants>DEBUG;TRACE</DefineConstants>
>      <ErrorReport>prompt</ErrorReport>
>      <WarningLevel>4</WarningLevel>
> +    <PlatformTarget>x86</PlatformTarget>

Do we need this?

---
Reply to this email directly or view it on GitHub:
https://github.com/Shazwazza/UmbracoLinqPadDriver/pull/1/files#r28285671